packages/obojobo-express/migrations/20180928152658-create-binaries-table.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+var dbm
+var type
+var seed
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+	dbm = options.dbmigrate
+	type = dbm.dataType
+	seed = seedLink
+}
+
+exports.up = function(db) {
+	return db
+		.createTable('binaries', {
+			id: { type: 'UUID', primaryKey: true, defaultValue: new String('uuid_generate_v4()') },
+			created_at: {
+				type: 'timestamp WITH TIME ZONE',
+				notNull: true,
+				defaultValue: new String('now()')
+			},
+			blob: { type: 'bytea', notNull: true },
+			file_size: { type: 'int', notNull: true },
+			mime_type: { type: 'varchar', length: 25, notNull: true }
+		})
+		.then(result => {
```

Comment:
can change `.then`'s that do nothing but return to be:
`.then(() => db.addIndex('binaries', 'binaries_created_at_index', ['created_at'])`
Also, since the `result` is not used we can omit it as a parameter.

Just left a single comment here, but this applies to all occurrences in the code.

Reply:
Fixed in commit 925399d8f24a6b08d005a8023ce109b6c1d9ab36